File: SDKs/github.com/ghes-3.0/1.1.4/python/src/sdk/models/operations/actions_get_self_hosted_runner_group_for_org.py
from dataclasses import dataclass, field
from typing import Optional
from sdk.models import shared


@dataclass
class ActionsGetSelfHostedRunnerGroupForOrgPathParams:
    org: str = field(default=None, metadata={'path_param': { 'field_name': 'org', 'style': 'simple', 'explode': False }})
    runner_group_id: int = field(default=None, metadata={'path_param': { 'field_name': 'runner_group_id', 'style': 'simple', 'explode': False }})
    

@dataclass
class ActionsGetSelfHostedRunnerGroupForOrgRequest:
    path_params: ActionsGetSelfHostedRunnerGroupForOrgPathParams = field(default=None)
    

@dataclass
class ActionsGetSelfHostedRunnerGroupForOrgResponse:
    content_type: str = field(default=None)
    status_code: int = field(default=None)
    runner_groups_org: Optional[shared.RunnerGroupsOrg] = field(default=None)
    

File: SDKs/github.com/ghes-3.0/1.1.4/typescript/src/sdk/models/operations/actionsgetselfhostedrunnergroupfororg.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";
import * as shared from "../shared";


export class ActionsGetSelfHostedRunnerGroupForOrgPathParams extends SpeakeasyBase {
  @Metadata({ data: "pathParam, style=simple;explode=false;name=org" })
  org: string;

  @Metadata({ data: "pathParam, style=simple;explode=false;name=runner_group_id" })
  runnerGroupId: number;
}


export class ActionsGetSelfHostedRunnerGroupForOrgRequest extends SpeakeasyBase {
  @Metadata()
  pathParams: ActionsGetSelfHostedRunnerGroupForOrgPathParams;
}


export class ActionsGetSelfHostedRunnerGroupForOrgResponse extends SpeakeasyBase {
  @Metadata()
  contentType: string;

  @Metadata()
  statusCode: number;

  @Metadata()
  runnerGroupsOrg?: shared.RunnerGroupsOrg;
}
